src/System.Private.CoreLib/shared/System/Threading/Tasks/Task.cs

```diff
@@ -882,21 +900,10 @@ internal bool MarkStarted()
             return AtomicStateUpdate(TASK_STATE_STARTED, TASK_STATE_CANCELED | TASK_STATE_STARTED);
         }
 
-        internal bool FireTaskScheduledIfNeeded(TaskScheduler ts)
+        [Conditional("CORECLR")]
```

Comment:
Why is this needed?

Reply:
It's not, I'm just not sure if CoreRT runtime can remove the call